pippo-core/src/main/java/ro/pippo/core/PippoFilter.java

```diff
@@ -107,12 +111,16 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         // TODO test for redirect
         // no redirect; process the request
 
-        String requestMethod = httpServletRequest.getMethod();
+        // create Request, Response objects
```

Comment:
I see that `getRelativePath` method is not used anymore. Now we create the `Request`, `Response` instances in advance to use `Request.getApplicationRequestPath()` but are cases when the relativePath is ignored so we don't need the Request, Response instances.